clientgui/ProjectInfoPage.cpp

```diff
@@ -42,10 +42,14 @@
 #include "res/windowsicon.xpm"
 #include "res/macosicon.xpm"
 #include "res/linuxicon.xpm"
+#include "res/androidicon.xpm"
 #include "res/freebsdicon.xpm"
+#include "res/linuxarmicon2.xpm"
 #include "res/atiicon.xpm"
+#include "res/amdicon2.xpm"
```

Comment:
Merge ATi and AMD here and in the rest of the code.

Reply:
Code has to handle plan class names using either convention.

Reply:
I had misunderstood your intentions. I thought you were going to have ATi and AMD icons both which would not have made much sense.